projects/adrv9009/src/app/headless.c

```diff
@@ -30,6 +30,9 @@
 #include "app_jesd.h"
 #include "app_transceiver.h"
 #include "app_talise.h"
+#ifndef ALTERA_PLATFORM
+#include "xil_cache.h"
+#endif
```

Comment:
Isn't this an error (the subject of this commit states that some warnings are fixed)? Anyway, I would split this commit into multiple patches and describe exactly what is wrong.

Reply:
You're right, I will commit separately